front/src/Components/Profile.jsx

```diff
@@ -11,38 +12,31 @@ class Profile extends React.Component {
 			picture: '',
 			biography: '',
 			ville: '',
-			tags: []
+			posts: []
 		}
 	}
 	componentDidMount() {
 		const url = [
 			`http://localhost:3000/profiles/${this.props.id}`,
-			'http://localhost:3000/tags'
+			`http://localhost:3000/profiles/${this.props.id}/posts`
```

Comment:
are you sure it isn't `http://localhost:3000/profiles/${this.props.match.params.id}/posts` ?

Reply:
you are right, corrected. Thanks, it will be easier later